src/lib.rs

```diff
@@ -1,4 +1,10 @@
+mod convert;
 mod crc;
+pub mod ed25519;
```

Comment:
I like the rearrangement of ed25519 under a module instead of including that in the type names. 👏🏻